wurst/network/Network.wurst

```diff
@@ -0,0 +1,506 @@
+package Network
+
+import SyncSimple
+import ErrorHandling
+import Execute
+import Buffer
+
+import initlater Metadata
```

Comment:
You should avoid initlater if possible. I think for the most parts you just access constants from another package.
I would say this means either move those into a config package or move the init to the other package

Reply:
Moved config out into separate package